cnxauthoring/tests/test_functional.py

```diff
@@ -2799,6 +2799,24 @@ def test_user_contents_hide_documents_inside_binders(self):
                 },
             })
 
+    def test_storage_unavailable_error(self):
+        '''
+        Test API error responce when POST is called during a closed
+        or lost database connection.
+        '''
+        import psycopg2
+        from ..storage import storage
+
+        storage.conn.close()
```

Comment:
You probably want to do `self.addCleanup(psycopg2.connect, storage.conn.dsn)`.  In case anything in this test fails, the clean up code will still get run.